themes/default/content/what-is/resolve-unable-to-locate-credentials.md

```diff
@@ -0,0 +1,110 @@
+
+## Getting started with esc run
+
+### Step 1: Install and login to Pulumi ESC
+
+To begin, you’ll need to [install Pulumi ESC](/docs/install/esc/). Once the installation is complete, run the `esc login` command and follow the steps to login to the CLI.
+
+```bash
+$ esc login
+Manage your Pulumi ESC environments by logging in.
+Run `esc --help` for alternative login options.
+Enter your access token from https://app.pulumi.com/account/tokens
+    or hit <ENTER> to log in using your browser                   :  
+Logged in to pulumi.com as …
+```
+
+### Step 2: Create the OIDC configuration
+
+Pulumi ESC offers you the ability to manually set your credentials as secrets in your Pulumi ESC environment files. When it comes to something like OIDC configuration, a more secure and efficient alternative is to leverage yet another great feature of Pulumi ESC: dynamic credentials.
+
+This service can dynamically generate credentials on your behalf each time you need to interact with your AWS environments. To do so, follow the steps in the [guide for configuring OIDC between Pulumi and AWS](/docs/pulumi-cloud/oidc/aws/). Make sure that the IAM role you create has sufficient permissions to perform S3 actions.
+
+### Step 3: Create a new Pulumi ESC environment
+
+Once OIDC has been configured between Pulumi and AWS, the next steps is to create a new environment in the [Pulumi Cloud](https://app.pulumi.com/). Make sure that you have the correct organization selected in the left-hand navigation menu. From there, click the **Environments** link, then click the **Create environment** button. In the following pop-up, provide a name for your environment before clicking the **Create environment** button.
+
+{{< video title="Open environment in Pulumi ESC console" src="/what-is/esc-create-new-env.mp4" autoplay="true" loop="true" >}}
+
+### Step 4: Add the AWS provider integration
+
+Once you’ve created your new environment, you will be presented with a split-pane editor view. You’ll want to clear out the default placeholder content in the editor on the left-hand side and replace it with the following code, making sure to replace <your-oidc-iam-role-arn> with the value of your IAM role ARN from the configure OIDC step:
+
+```yaml
+values:
+  aws:
+    login:
+      fn::open::aws-login:
+        oidc:
+          duration: 1h
+          roleArn: <your-oidc-iam-role-arn>
+          sessionName: pulumi-environments-session
+  environmentVariables:
+    AWS_ACCESS_KEY_ID: ${aws.login.accessKeyId}
+    AWS_SECRET_ACCESS_KEY: ${aws.login.secretAccessKey}
+    AWS_SESSION_TOKEN: ${aws.login.sessionToken}
+```
+
+### Step 5: Run the aws CLI command
+
+With your environment set up, you can now validate your configuration. First make sure that your local environment does not have any AWS credentials configured. You can do this by running the `aws configure list` command as shown below:
```

Comment:
This is probably the only one that when you run this it should return blank i think. But you should update the sentence "First make sure that your local environment does not have any AWS credentials configured." should be updated to something like "because you get the error of unable to locate creds, when you run this command it should show up as below and blank" something like that

Reply:
@aaronkao I'm going to respond on this comment regarding this and all of the related comments. The goal here is to get them to clear out their credentials if any show up. I do recognize that there is a gap here in that this isn't clear. I think it would make more sense to include some "if creds present, then remove" text rather than personalize/customize every page at this section. "Make sure you dont have local credentials configured, if anything shows up here delete them and then run the esc run command" should be applicable/stampable to all pages regardless of the command.